jekyll/_cci2/credits.md

```diff
@@ -2,114 +2,37 @@
 layout: classic-docs
 title: Using Credits
 categories: [how-to]
-description: How to leverage CircleCI credits
+description: Learn about the CircleCI credits-based plan system
 version:
 - Cloud
 ---
 
-This document describes the basics of using credits with CircleCI. If you are on the legacy CircleCI Container-based plan you may want to consider consulting the document on [using containers]({{site.baseurl}}/2.0/containers). If you want to switch from using containers to using credits, please [open a support ticket](https://support.circleci.com/hc/en-us/requests/new) requesting so.
+This document describes how to find the available resources regarding credits with CircleCI. If you are on the legacy CircleCI Container-based plan you may want to consider consulting the document on [using containers]({{site.baseurl}}/2.0/containers). If you want to switch from using containers to using credits, please [open a support ticket](https://support.circleci.com/hc/en-us/requests/new) requesting so.
 
 ## Overview
 {: #overview }
 
-The CircleCI credit-based usage plans enable you to only pay for what you use, while also providing flexibility in customizing and scaling your team's CI solution. Credits are consumed by the second at varying rates according to what build configuration you use.
+Credits are used on CircleCI's Free, Performance, Scale, and Server plans. Each plan offers key CI/CD (continuous integration, continuous development) features, and some plans offer customization options depending on your needs.
```

Comment:
```suggestion
Credits are used on CircleCI's Free, Performance, Scale, and Server plans. Each plan offers key Continuous Integration, Continuous Development (CI/CD) features, and some plans offer customization options depending on your needs.
```

Reply:
The CD is "continuous deployment" I believe?

Reply:
Actually rather than spelling out CI/CD in docs can we just link to: https://circleci.com/continuous-integration/#what-is-continuous-integration, so leave it as CI/CD

I asked a few folks who said its a grey area, people take it to mean different things depending on whether they want delivery or deployment etc. I will add this to the style guide I think.